src/containers/App/App.js

```diff
@@ -5,30 +5,40 @@ import { connect } from 'react-redux';
 import { getMovies } from '../../actions';
 import API from '../../utils/api';
 import { apiKey } from '../../utils/api-key';
+import Movie from '../Movie/Movie'
 
 class App extends Component {
-  constructor(props) {
-    super(props);
-
-  }
 
   componentDidMount = async () => {
-    const url = `https://api.themoviedb.org/3/movie/now_playing?api_key=${apiKey}&language=en-US&page=1`;
+    const initialCategory = 'popular'
+    const root = `https://api.themoviedb.org/3/movie/${initialCategory}`
+    const url = `${root}?page=1&api_key=${apiKey}&language=en-US`
```

Comment:
I like how you made this dynamic if we want to change where we are pulling from in the future

Reply:
I think the initial category will turn into current category and be routed accordingly, but it's a place to start at least